spacy_llm/tasks/ner.py

```diff
@@ -92,12 +90,91 @@ class NERTask:
 Text:
 '''
 {{ text }}
+'''
+"""
+
+
+@registry.llm_tasks("spacy.NER.v1")
+def make_ner_task_v1(
+    labels: str,
+    examples: Optional[Callable[[], Iterable[Any]]] = None,
+    normalizer: Optional[Callable[[str], str]] = None,
+    alignment_mode: Literal["strict", "contract", "expand"] = "contract",  # noqa: F821
+    case_sensitive_matching: bool = False,
+    single_match: bool = False,
+):
+    task = NERTask(
+        labels=labels,
+        examples=examples,
+        normalizer=normalizer,
+        alignment_mode=alignment_mode,
+        case_sensitive_matching=case_sensitive_matching,
+        single_match=single_match,
+    )
+    setattr(task, "_TEMPLATE_STR", _NER_TEMPLATE_V1)
```

Comment:
Why does this need to be `setattr`?

Reply:
And this feels like it ought to be an `NERTask` property instead maybe? (Where you'd set this from a template in `initialize` if you needed to? And where it should be saved with `llm.to_disk`? But that doesn't need to be in this PR.)

Reply:
Agreed, but see #81

Reply:
But why isn't it just `task._TEMPLATE_STR = _NER_TEMPLATE_V1`?

Reply:
oh pyright error in VSCode for me. But I just didn't really look at the error, fixed this